CMakeLists.txt

```diff
@@ -80,6 +80,14 @@ find_package(USD REQUIRED)
 include(cmake/usd.cmake)
 include(${USD_CONFIG_FILE})
 
+string(SUBSTRING ${MAYA_API_VERSION} "0" "4" MAYA_MAJOR_VERSION)
```

Comment:
I also wonder if instead of doing this here, I add these two version variables in FindMaya.cmake:

MAYA_MAJOR_VERSION
MAYA_MINOR_VERSION

Reply:
Yes I think this should be in FindMaya.cmake. But what I would suggest is to look at the section where it gets "MAYA_API_VERSION" from MTypes.h. I would suggest adding a similar regex to get the **MAYA_APP_VERSION**. That variable was added to MTypes.h starting in Maya 2019. So if you find it (in MTypes.h) you can use it directly. Otherwise you can use your logic above to read the MAYA_APP_VERSION from the first 4 chars of the API version.

Reply:
Is there any reason why we cannot test with MAYA_API_VERSION directly?

Reply:
@HdC-adsk it just easier to think about the MAYA_APP_VERSION and not worry about the zeros or any numbers that goes after:) 

e.g
#define MAYA_API_VERSION 20180000
#define MAYA_API_VERSION 20210000
vs
#define MAYA_APP_VERSION 2018
#define MAYA_APP_VERSION 2019

Reply:
I think it is easy to test with MAYA_API_VERSION < 20190000. I don't have strong preference, so I guess we will need to remove the "MAYA_APP_VERSION not allowed" item from coding standard. (I added it only because MAYA_API_VERSION is consistent with all recent releases.) What do you think?